e2e/packages/contracts/src/codegen/tables/NumberList.sol

```diff
@@ -194,9 +194,11 @@ library NumberList {
 
```

Comment:
do templates codegen need to be updated?

Reply:
double-checked, nope (I think CI'd catch it). This affects only tables with dynamic fields, which templates don't have